sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -42,12 +42,28 @@ To use this starter in a resource server, please add following packages:
     <version>3.0.0-beta.1</version>
 </dependency>
 <dependency>
-    <groupId>org.springframework.security</groupId>
-    <artifactId>spring-security-oauth2-client</artifactId>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-oauth2-resource-server</artifactId>
+</dependency>
+```
+[//]: # "{x-version-update-end}"
+
+To use this starter in a resource server with OBO function, please add following packages:
```

Comment:
The dependency is different from `resource-server` and `resource-server` with obo function.